application/microservices/entando-hub-catalog-ms/src/main/resources/config/changelog/20230329000000_private_catalog_api_key.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd
+                        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+    <!-- create private_catalog_api_key table -->
+    <changeSet id="20230329-private-catalog-api-key" author="admin">
+        <addUniqueConstraint columnNames="username" tableName="portal_user" constraintName="username_portal_user_unique_key"/>
+        <createTable tableName="private_catalog_api_key">
+            <column name="id" type="int" >
+                <constraints primaryKey="true" nullable="false" />
+            </column>
+            <column name="api_key" type="varchar(128)">
```

Comment:
Hi @antromeo I double checked the table structure generated from the jpa entity and this is a varchar(255) field. 
Could you please change it?

Reply:
sorry guys but JPA should not alter the database anymore since we are setting it with liquibase, no?